src/jdk.httpserver/share/classes/module-info.java

```diff
@@ -27,6 +27,12 @@
  * Defines the JDK-specific HTTP server API, and provides the jwebserver tool
  * for running a minimal HTTP server.
  *
+ * <p>The API, SPI and the server implementation in this module is intended
```

Comment:
1. Add a comma after SPI. ( I had to look up that acronym).
2. Change "is intended" to "are intended."

Reply:
Thank you Brian for these inputs. I have updated the PR to follow these suggestions.

> Add a comma after SPI. ( I had to look up that acronym).

That's a good point. The first usage of SPI was a few lines below the current proposed text. And on that first usage, it was fully spelled out as "Service Provider Interface (SPI)". I've now moved that part to this line and replaced that second usage with just SPI.